src/test/java/org/openmrs/contrib/qaframework/automation/AllergiesSteps.java

```diff
@@ -42,27 +42,28 @@ public void loadAllergiesPage() {
 
 	@Then("the system loads Allergies page")
 	public void systemLoadsAllergiesPage() {
-		assertEquals(getElement(patientHeaderId).getText(),getElement(patientHeaderId).getText());
-		assertTrue(textExists("Allergies"));
+		assertPage(allergyPage.waitForPage());
 	}
 
 	@And("a user clicks No Known Allergy button")
 	public void addNoKnownAllergy() {
 		allergyPage.addNoKnownAllergy();
+		allergyPage.waitForPage();
 	}
 
 	@Then("the system add no known allergies into the allergies table")
 	public void systemAddsNoKnownAllergies() {
-		assertTrue(textExists("No known allergies"));
```

Comment:
Havent seen a good reason for this refactor, Do you mind giving me a brief way why this change.

Reply:
Have been noticing random breakage of these automated tests on ci and opted to improve them as a way of ensuring they are stable and can serve the intended purpose otherwise if these tests are not stable then it will have less value as far as our vision is concerned. We need to have them stabilize and reliable on every build plan run !

@dkayiwa  emphasized this via slack where he said `this instability will make it unreliable to be of value`.

Reply:
> Havent seen a good reason for this refactor

assertPage() is more reliable than assertText() in reference to automated tests.

Reply:
Have been noticing random breakage of these automated tests on ci and opted to improve them as a way of ensuring they are stable and can serve the intended purpose otherwise if these tests are not stable then it will have less value as far as our vision is concerned. We need to have them stabilize and reliable on every build plan run !

@dkayiwa  emphasized this via slack where he said `this instability will make it unreliable to be of value`.